src/android/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.18.1)
 
+set(REACT_NATIVE_VERSION "0.71.0-rc.5")
```

Comment:
@RedBeard0531 should we capture an issue to get this derived? I also see it defined on line 7 of this file - any idea why it needed to go here too?